frontend/src/main/scala/bloop/engine/State.scala

```diff
@@ -97,4 +96,22 @@ object State {
     ExecutionContext.executor.setCorePoolSize(threads)
     state
   }
+
+  import bloop.Project
+  import bloop.io.AbsolutePath
+  def loadStateFor(
+      configDirectory: AbsolutePath,
+      commonOptions: CommonOptions,
+      logger: Logger
+  ): State = {
```

Comment:
I'm wondering if this shouldn't be renamed or refactored. It makes it look like we're loading a state for this `configDirectory`, `commonOptions` and `logger`...

Reply:
Let's call it `loadActiveStateFor`.